packages/ng/tag/tag.component.html

```diff
@@ -0,0 +1,15 @@
+@if (href) {
+<a [href]="href" class="tag mod-{{size}} palette-{{palette}}" [class.mod-outlined]="outlined">
+	@if (icon) {
+	<lu-icon [icon]="icon"></lu-icon><span class="tag-content">{{label}}</span>
+	} @else {
+	{{label}}
+	}</a>
+} @else {
+<span class="tag mod-{{size}} palette-{{palette}}" [class.mod-outlined]="outlined">
+	@if (icon) {
+	<lu-icon [icon]="icon"></lu-icon><span class="tag-content">{{label}}</span>
+	} @else {
+	{{label}}
+}</span>
+}
```

Comment:
I haven't tried control flow yet but it seems not to indent inside @if blocks
```suggestion
@if (href) {
	<a [href]="href" class="tag mod-{{size}} palette-{{palette}}" [class.mod-outlined]="outlined">
	@if (icon) {
		<lu-icon [icon]="icon"></lu-icon><span class="tag-content">{{label}}</span>
	} @else {
		{{label}}
	}</a>
} @else {
	<span class="tag mod-{{size}} palette-{{palette}}" [class.mod-outlined]="outlined">
	@if (icon) {
		<lu-icon [icon]="icon"></lu-icon><span class="tag-content">{{label}}</span>
	} @else {
		{{label}}
	}</span>
}
```